Library/Homebrew/cmd/install.rb

```diff
@@ -22,6 +22,7 @@ module Homebrew
 
   sig { returns(CLI::Parser) }
   def install_args
+    # rubocop:disable Metrics/BlockLength
```

Comment:
Please increase the limit rather than disabling here, thanks.

Reply:
👍